modules/con_ipi-installation-workflow.adoc

```diff
@@ -0,0 +1,48 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_bare_metal/ipi/ipi-install-prerequisites.adoc
+
+:_mod-docs-content-type: CONCEPT
+
+[id="installation-workflow_{context}"]
+= Installation workflow
+
+The installation program supports interactive mode, but it is recommended to prepare an `install-config.yaml` file in advance, containing the provisioning details for all of the bare-metal hosts.
+
+The administrator adds the relevant cluster details to the `install-config.yaml` file. The `install-config.yaml` file is shipped to the provisioning host. The administrator generates the manifests and verifies all prerequisites.
+
+The installation program enrolls all nodes in the cluster and starts the bootstrap VM. Metal platform components are started as `systemd` services, which have the following containers:
+
+* Ironic-dnsmasq: DHCP server responsible for handing over the IP addresses to the provisioning interface of various nodes on the provisioning network. Ironic-dnsmasq is only enabled when the `provisioningNetwork` is enabled.
+* Ironic-httpd: http server used to ship the images to the nodes.
+* Image-customization
+* Ironic
+* Ironic-inspector (available in {openshift} 4.16 and earlier)
+* Ironic-ramdisk-logs
+* Extract-machine-os
+* Provisioning-interface: Service that configures networking when the provisioning interface is enabled.
+* Metal3-baremetal-operator: Operator that provides a higher-level API for Ironic and replaces `terraform-provider-ironic` (available in {openshift} 4.16 and later)
+
+The nodes enter the validation phase. The nodes move to a “manageable” state after Ironic validates the credentials to access the Baseboard Management Controller (BMC) for each node.
```

Comment:
Consider "...phase, where each node moves..." and omit "for each node" at the end.